src/components/players/SunbirdVideoPlayer.tsx

```diff
@@ -26,7 +26,16 @@ const SunbirdVideoPlayer = ({ playerConfig }: PlayerConfigProps) => {
 
     const handlePlayerEvent = (event: any) => {
       console.log("Player Event", event.detail);
+      if (event?.detail?.type === "EXIT") {
+        const previousPage = sessionStorage.getItem("previousPage");
+        if (previousPage) {
```

Comment:
@suvarnakale try to reduce this duplicate code.

Reply:
@itsvick we are hadling each player separately so events needs to be handle separately also event data in different for different player. ex- event?.detail?.type === "EXIT" ; event?.detail?.edata?.type === "EXIT"  ;

Reply:
managing it from helper file